integrated_channels/sap_success_factors/models.py

```diff
@@ -52,6 +52,8 @@ class SAPSuccessFactorsEnterpriseCustomerConfiguration(TimeStampedModel):
     sapsf_base_url = models.CharField(max_length=255)
     key = models.CharField(max_length=255, blank=True, verbose_name="Client ID")
     secret = models.CharField(max_length=255, blank=True, verbose_name="Client Secret")
+    sapsf_company_id = models.CharField(max_length=255, blank=True, verbose_name="SAP Company ID")
```

Comment:
I'm not sure if alphabetizing the model fields results in actual migration file changes, but my OCD would love for them to be alphabetized.  Also do we need to add a verbose name to `sapsf_base_url`?